install/ontology/general.rdf

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<rdf:RDF
+        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+        xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+        xmlns:lom="http://www.tao.lu/Ontologies/TAOLom.rdf#"
+        xml:base="http://www.tao.lu/Ontologies/TAOLom.rdf#"
+        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
+        xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+>
+
+    <!--LOM properties for General category-->
+
+    <!--General identifier-->
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAOLom.rdf#general-identifier">
```

Comment:
I would state explicitly that the Identifier property is NOT language dependent. There is no "entailment rule" for that ;) !

(FYI, it will work, but for the sake of a clear ontology, let's go in a super clear description of the ontology)